opencga-app/app/scripts/azure/arm/azuredeploy.json

```
@@ -229,6 +273,82 @@
                 }
             }
         },
+        {
+            "comments": "This deploys a VM to run the OpenCGA daemon",
+            "resourceGroup": "[parameters('rgPrefix')]",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2018-05-01",
+            "name": "daemonvm",
+            "dependsOn": [
+                "azurefiles",
+                "vnet",
+                "[parameters('rgPrefix')]"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(parameters('_artifactsLocation'), '/daemonvm/azuredeploy.json', parameters('_artifactsLocationSasToken'))]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+
+                    "_artifactsLocation": {
+                        "value": "[parameters('_artifactsLocation')]"
+                    },
+                    "_artifactsLocationSasToken": {
+                        "value": "[parameters('_artifactsLocationSasToken')]"
+                    },
+                    "virtualNetworkId": {
+                        "value": "[reference('vnet').outputs.vnetId.value]"
+                    },
+                    "adminPassword": {
+                        "value": "[parameters('daemonVmAdminPassword')]"
+                    },
+                    "hdInsightSshDns": {
+                        "value":  "[reference('hdinsight').outputs.clusterSshDnsName.value]"
+                    },
+                    "hdInsightSshUsername": {
+                        "value": "[reference('hdinsight').outputs.clusterSshUsername.value]"
+                    },
+                    "hdInsightSshPassword": {
+                        "value":  "[reference('hdinsight').outputs.clusterSshPassword.value]"
+                    },
+                    "daemonContainerImage": {
+                        "value": "[parameters('daemonContainerImage')]"
+                    },
+                    "initContainerImage": {
+                        "value": "[parameters('initContainerImage')]"
+                    },
+                    "mountArgs": {
+                        "value": "[concat('azurefiles ', reference('azurefiles').outputs.accountName.value, ',', reference('azurefiles').outputs.shareName.value, ',' , reference('azurefiles').outputs.accountKey.value)]"

```

To pick up and use the switch from #980 you can change this line to switch between `avere` and `azurefiles` like this:

```suggestion
                         "value": "[if(equals(parameters('nfsStorageOption'), 'AzureFiles'), concat('azurefiles ', reference('azurefiles').outputs.accountName.value, ',', reference('azurefiles').outputs.shareName.value, ',' , reference('azurefiles').outputs.accountKey.value),concat('avere ', reference('avere').outputs.VSERVER_IPS.value))]"
```

---

thanks!